web/src/util/build.js

```diff
@@ -8,9 +8,11 @@ import config from '../app/config.js';
 const staticDir = config.staticDir;
 const require = createRequire(import.meta.url);
 // resolves to the node_modules folder with govuk-frontend and @ministryofjustice/frontend in
-const govUkNodeModules = path.resolve(require.resolve('govuk-frontend'), '../../../..');
+const govUkFrontend = path.resolve(require.resolve('govuk-frontend'), '../../../..');
 const mojNodeModules = path.resolve(require.resolve('@ministryofjustice/frontend'), '../../../..');
```

Comment:
The comment is out of date, and let's use the same suffix for govUk and moj how about:

- `govUkRoot` & `mojRoot` ?
- comment: 

```
// resolves to <root>/node_modules/govuk-frontend/govuk/all.js than maps to `<root>`
const govukRoot ...
// resolves to <root>/node_modules/@ministryofjustice/frontend/moj/all.js then maps to `<root>`
const mojRoot ...
```

Reply:
updated